SU-SWS/stanford_courses
# HSDO-636 #Created custom date filter for relative filtering with an exception timeframe

# To Test

1. Checkout branch HSDO-636-date-filter
2. Enable module stanford_course_views or Clear cache if already enabled.
3. Create/Edit a view with a date
4. Add/Edit a filter for a date field.
5. in the configuration form check the box "Add Exception Timeframe"
6. Configure the date window for the exception.
7. Configure the relative dates values.
8. Validate exception that is before current date, after current date, and current date is in exception window.
